src/app/modules/create-association/create-association.component.ts

```diff
@@ -0,0 +1,44 @@
+import { CreateAssociationService } from './services/create-association.service';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, Validators, FormArray } from '@angular/forms';
+
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-create-association',
+  templateUrl: './create-association.component.html',
+  styleUrls: ['./create-association.component.scss']
+})
+export class CreateAssociationComponent implements OnInit {
+
+  constructor(private fb: FormBuilder,
+    private createAssociationService: CreateAssociationService,
+    private router: Router) { }
+
+    profileForm = this.fb.group({
```

Comment:
fiind o variabila pana la urma.. nu ar trebui sa fie sub constructor nu? care e ordinea in care se scriu chestiile intr-o clasa?